jwst/tests/base_classes.py

```diff
@@ -59,16 +59,21 @@ def auto_toggle_docopy(self):
     def repo_path(self):
         return [self.inputs_root, self.env, self.input_loc]
 
-    def get_data(self, *pathargs, docopy=True):
+    def get_data(self, *pathargs, docopy=None):
         """
         Download `filename` into working directory using
         `artifactory_helpers/get_bigdata()`.
         This will then return the full path to the local copy of the file.
         """
-        # If user has specified action for no_copy, apply it with
+        # If user has specified action for docopy use it, If none
         # default behavior being whatever was defined in the base class.
-        local_file = get_bigdata(*self.repo_path, *pathargs, docopy=self.docopy)
-
+        if docopy is None: 
+            local_file = get_bigdata(*self.repo_path, *pathargs, docopy=self.docopy)
+        elif docopy:
+            local_file = get_bigdata(*self.repo_path, *pathargs, docopy=True)
+        else:
+            local_file = get_bigdata(*self.repo_path, *pathargs, docopy=False)
```

Comment:
Nice fix here, though I think we can be even a bit more ruthless.  I think we can just completely get rid of `self.docopy`.  I.e we can delete the fixture

https://github.com/spacetelescope/jwst/blob/0e282178154de761d09459ac8d94458e7ea14012/jwst/tests/base_classes.py#L50-L56

And have `get_data()` always use `docopy=True`.  Since we are hardcoding it in tests, there's no reason to have it be dynamically set in the test infrastructure.  Besides, I think the use case of `docopy=False` has disappeared.